liquibase-standard/src/test/groovy/liquibase/changelog/DatabaseChangeLogTest.groovy

```diff
@@ -473,14 +472,14 @@ http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbch
                 "include-all-dir/include-all.xml": changelogText,
         ])
         def changeLogFile = new DatabaseChangeLog("com/example/root.xml")
-        changeLogFile.includeAll("include-all-dir", false, null, true, changeLogFile.getStandardChangeLogComparator(), resourceAccessor, new ContextExpression(), new LabelExpression(), false)
+        changeLogFile.includeAll("include-all-dir", false, null, true, changeLogFile.getStandardChangeLogComparator(), resourceAccessor, new ContextExpression(), new Labels(), false, 0, Integer.MAX_VALUE)
 
         then:
         SetupException e = thrown()
         assert e.getMessage().startsWith("liquibase.exception.SetupException: Circular reference detected in 'include-all-dir/'. Set liquibase.errorOnCircularIncludeAll if you'd like to ignore this error.")
```

Comment:
I did not even realize there is a `errorOnCircularIncludeAll` parameter! This probably should also be added to the documentation when `errorIfMissing` is added.

CC @MalloD12 @rberezen

Reply:
Ups I didn't see this comment before of creating a PD ticket. I'll add a comment there so the guys can also include this when working on the created ticket.

Reply:
I think it's not needed. I just found it's documented here https://docs.liquibase.com/parameters/error-on-circular-include-all.html?Highlight=errorOnCircularIncludeAll

Reply:
I see; it's a global parameter and not a changeset property. My mistake. Thanks for the link to the documentation!